flask_security/core.py

```diff
@@ -26,6 +30,9 @@
     ForgotPasswordForm, ChangePasswordForm, ResetPasswordForm, \
     SendConfirmationForm, PasswordlessLoginForm
 
+# Select stack, in flask 0.9 _app_ctx_stack is used
```

Comment:
Maybe this will be more clear?

```python
try:
    from flask import _app_ctx_stack as _ctx_stack
except ImportError:
    from flask import _request_ctx_stack as _ctx_stack
```